package.json

```diff
@@ -61,6 +63,7 @@
     "eslint": "^8.40.0",
     "jest": "^29.5.0",
     "postcss": "^8.4.23",
+    "prettier": "^2.8.8",
```

Comment:
Lint/format checks use prettier, so must be a dev dependency

Reply:
I guess I have `pnpm` configured to auto-install peer deps. Thanks!